app-script-payslip-sender/lab1.gs

```diff
@@ -0,0 +1,33 @@
+function sendEmail() {
+  let excel = SpreadsheetApp.getActiveSpreadsheet();
+  let sheet = excel.getSheetByName("Sheet1");
+  // Getting last row
+  let lastRow  = sheet.getLastRow();
+  // Getting second last Column
+  let secondLastColumn = sheet.getLastColumn()-1
+  let range = sheet.getRange(2,2,lastRow-1,secondLastColumn)
+  let values = range.getValues();
+
+  for (let i = 0; i < values.length; i++) {
+    let status;
+    let [Name, Email, Salary] = values[i];
```

Comment:
Suggest to follow standard naming convention for JavaScript variable.

https://www.30secondsofcode.org/js/s/naming-conventions/